meson.build

```diff
@@ -0,0 +1,61 @@
+project('avvie',
+          version: '0.1.0',
+    meson_version: '>= 0.59.0',
+  default_options: [ 'warning_level=2',
+                   ],
+)
+
+i18n = import('i18n')
+
+gnome = import('gnome')
+
+python = import('python')
+
+
+message('Looking for dependencies')
+python_bin = python.find_installation('python3')
+if not python_bin.found()
+    error('No valid python binary found')
+else
+    message('Found python binary')
+endif
```

Comment:
find_installation defaults to `required: true` and therefore it is impossible to hit the "not" case here as Meson will immediately error out on line 16 without progressing to line 17.